waspc/e2e-test/ShellCommands.hs

```diff
@@ -71,11 +72,14 @@ createFile content relDirFp filename = return $ combineShellCommands [createPare
 -- NOTE: This is fragile and will likely break in future. Assumes `app` decl is first line and by default
 --       we do not have a `db` field. Consider better alternatives.
 setDbToPSQL :: ShellCommandBuilder ShellCommand
-setDbToPSQL =
+-- Change DB to postgres by adding string at specific line so it still parses.
+setDbToPSQL = insertLineAtWaspFile "  db: { system: PostgreSQL }," 2
+
+insertLineAtWaspFile :: String -> Int -> ShellCommandBuilder ShellCommand
+insertLineAtWaspFile line atLineNumber =
```

Comment:
Nice extraction here 👍🏻

Reply:
In the future, is the plan to still possibly do the wasp file manipulation in memory in Haskell? Where there any insights/blockers you thought of for the future on that one we may want to document now while still fresh?

Reply:
Even manipulating it as a in-memory string will be an improvement :)

@shayneczyzewski and I started talking about doing it because I thought I had some concurency issues and some things got written to the `.wasp` file and some didn't. I thought that's due to doing the file manipulation via shell commands.

But that wasn't the casue, I had a bug in a different place (running `wasp compile` in the middle of the process) and that messed things up.

Things work as expected, `.wasp` file is consturcted properly even with shell commands. That's why I decided to keep the improvement of doing it in-memory for some future PR :)